src/builtins/BuiltinIntl.cpp

```diff
@@ -972,7 +982,7 @@ void GlobalObject::installIntl(ExecutionState& state)
     m_intlDateTimeFormatPrototype = m_intlDateTimeFormat->getFunctionPrototype(state).asObject();
     m_intlDateTimeFormatPrototype->setGlobalIntrinsicObject(state, true);
 
-    FunctionObject* formatFunction = new NativeFunctionObject(state, NativeFunctionInfo(strings->getFormat, builtinIntlDateTimeFormatFormatGetter, 0, NativeFunctionInfo::Strict));
+    FunctionObject* formatFunction = new NativeFunctionObject(state, NativeFunctionInfo(state.context()->staticStrings().getFormat, builtinIntlDateTimeFormatFormatGetter, 0, NativeFunctionInfo::Strict));
```

Comment:
Why do you get a staticStrings reference again?

Reply:
Good advice